src/backend/distributed/sql/udfs/citus_nodes/latest.sql

```diff
@@ -0,0 +1,45 @@
+CREATE VIEW citus.citus_nodes AS
+SELECT
+  nodename AS "Host",
+  nodeport AS "Port",
+  CASE WHEN groupid = 0 THEN 'coordinator' ELSE 'worker' END AS "Node Type",
+  pg_size_pretty(nullif(citus_node_database_size(nodename, nodeport),0)) AS "Database Size",
+  (SELECT
+     count(*)
+   FROM
+     pg_dist_partition t
+   JOIN
+     pg_dist_shard s
+   USING
+     (logicalrelid)
+   JOIN
+     pg_dist_placement p
+   USING
+     (shardid)
+   WHERE p.groupid = n.groupid AND t.partmethod <> 'n'
+  ) AS "Distributed Table Shards",
+  (SELECT
+     count(*)
+   FROM
+     pg_dist_partition t
+   JOIN
+     pg_dist_shard s
+   USING
+     (logicalrelid)
+   JOIN
+     pg_dist_placement p
+   USING
+     (shardid)
+   WHERE p.groupid = n.groupid AND t.partmethod = 'n' AND t.repmodel = 't'
+  ) AS "Reference Tables",
+  round(100 * (1. - (ds.available_disk_size::double precision / nullif(ds.total_disk_size, 0)))) || '%' AS "Disk Usage"
```

Comment:
what about `PGDATA Disk usage`? as currently it is not easy to understand without looking at code what this disk usage refers to